### ISSUE TITLE
508-defect-2 [COGNITION]: Links MUST NOT rely on color

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18442]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Details

Links must not rely on color. Underlines are provided only on hover, which is not available to mobile users.

## Acceptance Criteria

- [ ] Links do not rely on color to convey that they can be interacted with 

## Proposed Solution (if known)

[As discussed in ticket #3556, these heading links should be underlined](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556).

## WCAG or Vendor Guidance (optional)

* [WCAG 1.4.1: Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)

## Screenshots or Trace Logs

<img width="678" alt="Screen Shot 2021-01-12 at 7 22 44 PM" src="https://user-images.githubusercontent.com/14154792/104390613-209c1300-550c-11eb-812f-5b3f7daa991a.png">

**Comment by zacharymorel** [comment]
@joshkimux What browser where you using to reproduce this issue with the links? This also was on mobile only?

**Comment by joshkimux** [comment]
@zacharymorel 
I used a greyscale filter before taking this screenshot to represent what it would look like to a color blind user! I should have clarified that in the ticket 🙇

**Comment by zacharymorel** [comment]
Sweet, thank you, perhaps @laurellawrence can give me some direction on an alternative for the links while the user is using greyscale?

**Comment by joshkimux** [comment]
@zacharymorel  I worked with Laurel on this same thing in another ticket! [Here it is if you'd like to reference it!](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556). Basically, just adding an underline permanently (as opposed to on hover) 😄

**Comment by zacharymorel** [comment]
Okay sweet. I can go with this! Thanks you.

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.

**Comment by MarciMcGuireGCIO** [comment]
Sprint planning note - this should be part of the larger UI consistency effort.

**Comment by MarciMcGuireGCIO** [comment]
@callen2563 can you confirm that the new designs take into account the need for underlined links on mobile? TIA?

**Comment by MarciMcGuireGCIO** [comment]
This will be resolved with the implementation of https://github.com/department-of-veterans-affairs/va.gov-team/issues/22452

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO Can we go about closing this soon?

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel as soon as we have the Staging review and the accessibility confirms its not an issue in the new UI design, I plan to close it.
